server/db/models/index.js

```diff
@@ -19,7 +19,7 @@ const Boat = require('./boat');
 Boat.belongsToMany(Order, { through: 'OrderItems' });
 Order.belongsToMany(Boat, { through: 'OrderItems' });
 
-// User.hasMany(Order);
+User.hasMany(Order);
```

Comment:
@JoshuaSkootsky to confirm, this enables the magic method
User.getOrders()

Reply:
that sounds like something we would want!